### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#221]
The icons on the Contact page should be: alt=&quot;&#039;

**Comment by dialsunny** [comment]
Applied in Production.
